website/docusaurus.config.js

```diff
@@ -566,7 +566,7 @@ module.exports = {
                 // more info at https://github.com/rdilweb/docusaurus-plugin-remote-content#options
                 name: 'content-sdks',
                 sourceBaseUrl: 'https://raw.githubusercontent.com/Unleash/', // gets prepended to all of the documents when fetching
-                outDir: 'docs/reference/sdks', // the base directory to output to.
+                outDir: 'docs/generated/sdks', // the base directory to output to.
```

Comment:
We _might_ consider changing this line to:

```suggestion
                outDir: 'docs/generated/reference/sdks', // the base directory to output to.
```

But I also think that's something we can do later if we need to. No need create an arbitrarily nested folder structure if we're not gonna need it.